gensrc/proto/olap_file.proto

```diff
@@ -66,6 +66,13 @@ enum SegmentsOverlapPB {
 message RowsetTxnMetaPB {
     repeated uint32 partial_update_column_ids = 1;
     repeated uint32 partial_update_column_unique_ids = 2;
+
+    message FooterInfo {
```

Comment:
It seems to be redundant with PagePointerPB.

Reply:
Footer is not a page or it is?

Reply:
FooterPointerPB